genesis_core/user_api/iam/api/controllers.py

```diff
@@ -47,6 +88,27 @@ def enforce(self, rule, do_raise=False, exc=None):
         return iam.enforcer.enforce(rule, do_raise, exc)
 
 
+class AltchaMixin:
+
+    def check_captcha(self, payload: str = None):
```

Comment:
We already have `self` here, why don't get `self.request.headers.get("payload")` inside and make this method usage easier, i.e. without always copy-pasted arguments?